charts/pokt-portal-v2/templates/deployment.yaml

```diff
@@ -73,19 +73,24 @@ spec:
             successThreshold: {{ .Values.middleware.startupProbe.successThreshold | default 1 }}
           {{- end }}
           volumeMounts:
-          {{- if .Values.externalSecrets.enabled }}
-          - mountPath: "/config/"
-            name: external-secrets
-            readOnly: true
-          {{- else }}
-          - mountPath: /config
+          - mountPath: {{ .Values.middleware.configMountPath | default "/config" }}
             name: config
-          {{- end }}
           env:
             - name: CONFIG_FILE
-              value: "/config/config.json"
```

Comment:
For now I am just opting to take a local config.json file and template into a configMap. 

Later, we can add a templated version for secret injection with external-secrets templating engine.